### ISSUE TITLE
Possibility to comfortably add system and version info to a bug report (which does not throw an own window)

### ISSUE THREAD

**Issue by mischlrebl** [greenshot/greenshot#378]
Extend the About dialog with some convenient features (all information are already there):

- Log (like pressing 'L')
- Info (like pressing 'i')
- Version and System info (like in the error report)

![image](https://user-images.githubusercontent.com/28529799/153006391-cbda82e5-5729-489e-b2f6-8f58324d1e68.png)

**Comment by Lakritzator** [comment]
I fully agree that we need to add something, but I am not sure that I would like it there...
